apps/nextjs-website/.storybook/main.ts

```diff
@@ -0,0 +1,28 @@
+import type { StorybookConfig } from "@storybook/nextjs";
+
+import { join, dirname } from "path";
+
+/**
+ * This function is used to resolve the absolute path of a package.
+ * It is needed in projects that use Yarn PnP or are set up within a monorepo.
```

Comment:
According to this comment, we don't need this function, do we?
We are not using `Yarn`